FCCee/IDEA/compact/IDEA_o1_v03/MuonSystem_o1_v01.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd>
+
+  <info name="MuonSystem"
+	title=" detailed version of IDEA muon system for FCC-ee"
+	author="Mahmoud Ali, mahmoud.ali@cern.ch"
+	url="no"
+	status="development"
+	version="2.0">
+    <comment> It depends on the factory: muonSystemMuRWELL_o1_v01 </comment>
+  </info>
+
+  <define>
+     <!---                            Muon System Parameters                       -->
+     <!-- %%%%%%                      microRWELL chamber different layers thicknesses                %%%%%% -->
+       <constant name = "G10_FR4Thick"                          value = "1.6*mm"/>
+       <constant name = "CuThick"                               value = "0.035*mm"/>
+       <constant name = "GasLayerThick"                         value = "6*mm"/>
+       <constant name = "Cu2Thick"                              value = "0.005*mm"/>
+       <constant name = "KaptonThick"                           value = "0.05*mm"/>
+       <constant name = "CarnonFiberThick"                      value = "0.0001*mm"/>
+       <constant name = "CarbonFiber2Thick"                     value = "0.1*mm"/>
+       <constant name = "SiThick"                               value = "1.6*mm"/>
+       <constant name = "mRWELLTotalThickness"                  value = "G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick+CarbonFiber2Thick+CuThick+SiThick"/> <!-- This sequense is the current order of mRWELL (total 10 slices) -->
+
+       <constant name = "mRWELLYLength"                         value = "250*mm"/>
+       <constant name = "mRWELLZLength"                         value = "250*mm"/>
+
+       <constant name = "overlapingY"                           value = "5*mm"/> <!-- the common distance between mRWELL chambers in Y direction -->
+       <constant name = "overlapingZ"                           value = "5*mm"/> <!-- the common distance between mRWELL chambers in Z direction -->
+       <constant name = "clearance"                             value = "1*mm"/> <!-- it's a small distance to be used to avoid overlapping between the different volumes ~ 1 mm -->
+     <!-- %%%%%%                      Iron yoke thickness                %%%%%% -->
+      <constant name = "YokeThickness"                          value = "300*mm"/>  
+  </define>   
+
+  <readouts>
+    <readout name="MuonSystemCollection">
+     <segmentation type="CartesianGridYZ" grid_size_y="1.2*mm" grid_size_z="1.2*mm"/>   <!-- Depending on strip pitch 1.4 mm  -->
+     <id>system:5,type:2,layer:4,chamber:15,slice:1,y:-10,z:-10</id> <!-- The bit field is divided into 2^5 systems(IDEA sub-detectors), 2^2 types(Barrel Muon System, positive endcap, and negative endcap"), 2^4 layers(number of layers in barrel for example) ,2^15 chambers(the number of muRWELL chambers in every layer), 2^1 slice(number of sensitive layers inside every chambers), and 2^10 y&z strips in every sensitive layer-->
```

Comment:
```suggestion
     <id>system:5,type:2,layer:4,chamber:15,slice:1,y:-10,z:-10</id> <!-- The bit field is divided into 2^5 systems(IDEA sub-detectors), 2^2 types(0 = Barrel Muon System, +1 = positive endcap and -1 = negative endcap"), 2^4 layers(number of layers in barrel for example) ,2^15 chambers(the number of muRWELL chambers in every layer), 2^1 slice(number of sensitive layers inside every chambers), and 2^10 y&z strips in every sensitive layer-->
```?
I'd also propose to change `type` to `side`, but it is not a strong request.

Reply:
I'm afraid using `side` instead of `type` would cause confusion with the detector side variable that I use to determine the polygon shape sides.

Reply:
Ok! But can you still say in the xml which value is given to barrel/endcap etc?

Reply:
Yes, I just added it in the last commit